src/run/runner/wall_time/executor.rs

```diff
@@ -18,7 +18,9 @@ pub struct WallTimeExecutor {
 impl WallTimeExecutor {
     pub fn new() -> Self {
         let use_perf = if cfg!(target_os = "linux") {
-            std::env::var("CODSPEED_USE_PERF").is_ok()
+            std::env::var("CODSPEED_PERF_ENABLED")
```

Comment:
let's move this to a util function that takes the env var name as argument

and we might as well go all in on truthy values

`"1", "true", "yes", "on", "enabled"`
I don't have others in mind